### DISCUSSION TITLE
Configuration File Choice... pyproject.toml or bumpversion.toml?

### DISCUSSION THREAD

**Discussion by salderma** [callowayproject/bump-my-version#154] [Q&A] [answered]
Hi, I am new to this tool.  I've been a fan of using Poetry in my python projects, but a colleague suggested this tool for my current project which is a multi-language library (e.g. one that can be imported into python or node.js) in a single project.  So, I will be publishing/releasing two platform packages with each new version.  The python side of the project uses poetry for dependencies, testing, managing environments and the `pyproject.toml` file.  Is it generally safe to setup bump-my-version to modify the version tag in the `pyproject.toml` file?  Or should I have chosen the `pyproject.toml` file instead of `.bumpversion.toml` - I chose the latter since bump-my-version will also be editing version numbers for the files related to the node code.

Thanks for the community thoughts on my inquiry!

**Comment by coordt** [comment, marked as answer]
Either way works fine. You can modify the `pyproject.toml` even if your configuration is in it. Both options work the same way, so you are fine with your choice.
